tests/desktop_web/test_products_join.py

```diff
@@ -2,8 +2,8 @@
 import sentry_sdk
 from urllib.parse import urlencode
 
-def test_checkout_products_join(desktop_web_driver, endpoints, random, batch_size, backend, sleep_length):
-    sentry_sdk.set_tag("pytestName", "test_checkout_products_join")
+def test_products_join(desktop_web_driver, endpoints, random, batch_size, backend, sleep_length):
+    sentry_sdk.set_tag("pytestName", "test_products_join")
```

Comment:
sweet!! no more misnamed tests

I think in the future it would also be nice to have a comment at the top of each test
describing what purpose it serves (e.g. this one would be "demo transaction spans to show AFTER fixing N+1 Q issue")

Reply:
ah good point. I'll get that done (at some point)